.github/workflows/chart-release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release Charts
+on:
+  push:
+    tags:
+      - "*"
+jobs:
+  chart-release:
+    name: Chart Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Install Helm
+        uses: azure/setup-helm@v4
+        env:
+          GITHUB_TOKEN: "${{ secrets.RELEASE_GITHUB_TOKEN }}"
```

Comment:
Do we need the token for install helm?

Reply:
Good catch! The token is no longer needed. It was previously required to fetch the latest version, but that's not the case anymore. 
Azure/setup-helm#130.